sageopt/tests/test_coniclifts/test_problem.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+import numpy as np
+import time
+from sageopt.coniclifts.standards import constants as CL_CONSTANTS
+from sageopt.coniclifts.problems.solvers.ecos import ECOS
+from sageopt.coniclifts.problems.solvers.mosek import Mosek
+from sageopt.coniclifts.compilers import compile_problem
+from sageopt.coniclifts.base import Expression, Variable
+from sageopt.coniclifts import Problem, Constraint
+import sageopt.coniclifts as cl
+
+class TestProblem(unittest.TestCase):
+    def test_solve(self):
+        # random problem data
+        G = np.random.randn(3, 6)
+        h = G @ np.random.rand(6)
+        c = np.random.rand(6)
+        # input to coniclift's Problem constructor
+        x = cl.Variable(shape=(6,))
+        constrs = [0 <= x, G @ x == h]
+        objective_expression = c @ x
+        prob = cl.Problem(cl.MIN, objective_expression, constrs)
+        Problem.solve(prob, None)
+        # self.assertRaises(RuntimeError, Problem.solve, prob, None)
```

Comment:
Is there a particular behavior this test is getting at? I think it might be subsumed by tests in ``sageopt/tests/test_coniclifts/test_system/test_toys_1.py`` (where several simple problems are constructed and solved).

Reply:
I think I was originally intending to test some of the uncovered lines, but yes the solve function is already tested adequately in the location you mentioned.